pt/lessons/advanced/otp-supervisors.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: page
+title: Supervisores OTP
+category: advanced
+order: 6
+lang: pt
+---
+
+Supervisores são processos especializados com uma proposta: monitorar outros processos. Estes supervisores nos possibilitam a criação de aplicações tolerantes a falhas automaticamente reiniciando processos filhos quando eles falham.
```

Comment:
`propósito` usually used together with something after ex: `com o propósito de estabilizar a situação`. Following the english translation I think `proposta` is good.